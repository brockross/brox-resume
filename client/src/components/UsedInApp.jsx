import React from 'react';

const UsedInApp = (props) => {
  return (
    <div>
      <h3>Used in This App:</h3>
      <ul>
        <li>React</li>
        <li>Node</li>
        <li>Webpack</li>
        <li>RESTful APIs</li>
        <li>MongoDB</li>
      </ul>
    </div>
  )
}

export default UsedInApp;