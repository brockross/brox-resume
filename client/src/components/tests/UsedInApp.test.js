import React from 'react';
import { shallow } from 'enzyme';

import UsedInApp from  '../UsedInApp.jsx';

describe('UsedInApp', () => {
  it('should render correctly', () => {
    const component = shallow(<UsedInApp/>);

    expect(component).toMatchSnapshot();
  });
});