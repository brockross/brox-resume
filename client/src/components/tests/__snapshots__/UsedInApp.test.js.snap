// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UsedInApp should render correctly 1`] = `
<div>
  <h3>
    Used in This App:
  </h3>
  <ul>
    <li>
      React
    </li>
    <li>
      Node
    </li>
    <li>
      Webpack
    </li>
    <li>
      RESTful APIs
    </li>
    <li>
      MongoDB
    </li>
  </ul>
</div>
`;
